### ISSUE TITLE
add BufferSubdivisionModifier

### ISSUE THREAD

**Issue by flowen** [DelvarWorld/three-subdivision-modifier#1]
hi Andrew! Thanks so much for adding this package to npm. Perhaps you could also add the buffer version? THREE.BufferSubdivisionModifier

[see example](https://threejs.org/examples/?q=sub#webgl_modifier_subdivision)

**Comment by Laubeee** [comment]
Yeah, now would be a good time to do so, since Geometry got thrown out
has anyone already done it?
